PlatformIO/platformio.ini

```diff
@@ -44,7 +44,7 @@ lib_deps =
 
 [env:esp32dev]
 
-;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4
+;supported: M5ATOMECHO=0, MATRIXVOICE=1, AUDIOKIT=2, INMP441=3, INMP441MAX98357A=4, ESP32_POE_ISO=5, TAUDIO=6 INMP441MAX98357AFASTLED=7
```

Comment:
I think you missed a comma between the last two devices

Reply:
fixed